setup.cfg

```diff
@@ -1,10 +1,219 @@
-[aliases]
-build = update_version build
-sdist = update_version sdist
-install = update_version install
-develop = update_version develop
-bdist_egg = update_version bdist_egg
-bdist_wheel = update_version bdist_wheel
+# https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#declarative-config
+[metadata]
+name = tahoe-lafs
+
+[options]
+description = secure, decentralized, fault-tolerant file store
+long_description = file: README.rst
+author = the Tahoe-LAFS project
+author_email = tahoe-dev@lists.tahoe-lafs.org
+project_urls =
+    website=https://tahoe-lafs.org/
+    documentation=https://tahoe-lafs.readthedocs.org/
+    source=https://github.com/tahoe-lafs/tahoe-lafs
+classifiers = file: CLASSIFIERS.txt
+license_files = COPYING.GPL, COPYING.TGPPL.rst
+# tell setuptools to find our package source files automatically
+packages = find:
+# find packages beneath the src directory
+package_dir=
+    =src
+include_package_data = True
+
+# We support Python 3.7 or later. 3.11 is not supported yet.
```

Comment:
Are there problems with 3.11 or we just haven't tested..?

Reply:
Just haven't tested.

Reply:
Actually, I know that Twisted's iocpsupport module hasn't been published for Python 3.11 yet so if someone tries to install Tahoe-LAFS on Windows (and we didn't prohibit that here) then chances are it will fail with an error about Visual Studio being missing.